compiler/optimizer/UseDefInfo.cpp

```diff
@@ -2215,14 +2218,14 @@ void TR_UseDefInfo::dereferenceDef(TR_UseDefInfo::BitVector &useDefInfo, int32_t
    {
    TR_ASSERT(defIndex < getTotalNodes(), "TR_UseDefInfo::getNode index(%d) is bigger than total(%d)\n", defIndex, getTotalNodes());
 
-   CS2::LinkedListOf<CS2::Pair<TR::Node *,TR::TreeTop *>, TR::Allocator> defIndices(comp()->allocator());
-   defIndices.Push(_atoms[defIndex]);
+   TR::list<std::pair<TR::Node *,TR::TreeTop *>, TR::Region&> defIndices(_region);
```

Comment:
This likely would perform better as a deque, or even possibly a vector if we swapped to using push_back / back.  Though it may not enough use to matter.